lib/active_interaction/filter.rb

```diff
@@ -90,15 +90,16 @@ def initialize(name, options = {}, &block)
     #   # => ActiveInteraction::InvalidDefaultError: example: 0
     #
     # @param value [Object]
+    # @param interaction [Base, nil]
     #
     # @return [Object]
     #
     # @raise (see #cast)
     # @raise (see #default)
-    def clean(value)
-      value = cast(value)
+    def clean(value, interaction)
```

Comment:
Never mind. Using a binding won't work because then `self.foo` would mean `interaction.binding.foo` instead of `interaction.foo`.